### ISSUE TITLE
Change Footer

### ISSUE THREAD

**Issue by viviurbano** [viviurbano/wddm-120-create-and-read-app#6]
Change Footer, letting data better organized and structured

**Comment by rashmod** [comment]
Do you have a design that can be followed to update the footer?

**Comment by viviurbano** [comment]
@rashmod Yes, I have. Actually I am going to add Bootstrap for React (https://react-bootstrap.github.io/) in this project and use their model as a footer.
